Update docker, add methods to load from dictionary, get tree traversa… (#174)
networktocode/diffsync: docs/source/getting_started/01-getting-started.md

![Preorder Tree Traversal](../../images/preorder-tree-traversal.drawio.png "Preorder Tree Traversal")

### Mapping Tree Traversal with `get_tree_traversal` method

> For your convenience, there is a helper method that will provide a mapping of the order. The `DiffSync.get_tree_traversal` method will return a string, in a tree-like format or optionally a dictionary when setting the `as_dict` method on.

Reviewer: ```suggestion
For your convenience, there is a helper method that will provide a mapping of the order. The `DiffSync.get_tree_traversal()` method will return a tree-like string, or optionally a dictionary when passing the `as_dict=True` parameter.
```
